query/doc.html

```diff
@@ -84,8 +114,6 @@ <h2>Abstract</h2>
     </section>
 
     <section id="sotd" class="introductory">
-      <h2 id="status-of-this-document">Status of This Document
-        <a class="self-link" href="#sotd" aria-label="Permalink for Section"></a></h2>
       <p>
```

Comment:
Why is the h2 removed?

Reply:
"sotd" is a "magic" section, that automatically applies a heading. Adding one manually seems to change the way it renders.

Most sections automatically get permalink, although it seems that SOTD is not one of them. We can revert this, and I don't think it will matter too much.